spring-cloud-task-stream/src/main/java/org/springframework/cloud/task/batch/listener/EventEmittingItemProcessListener.java

```diff
@@ -64,4 +66,9 @@ else if (item.equals(result)) {
 	public void onProcessError(Object item, Exception e) {
 		messagePublisher.publishWithThrowableHeader("Exception while item was being processed", e.getMessage());
 	}
+
+	@Override
+	public int getOrder() {
+		return 0;
```

Comment:
If the below is OK i will proceed 

```
	@ConfigurationProperties(prefix = "spring.cloud.task.events")
	public class TaskEventsProperties {

		private ListenerProperties jobExecution;

		public ListenerProperties getJobExecution() {
			return jobExecution;
		}

		public void setJobExecution(ListenerProperties jobExecution) {
			this.jobExecution = jobExecution;
		}

		public  class ListenerProperties {

			private int order;

			public int getOrder() {
				return order;
			}

			public void setOrder(int order) {
				this.order = order;
			}
		}
	}
```

another issue the prefix should be `spring.cloud.task.events` or `spring.cloud.task.batch.events`